pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java

```diff
@@ -108,9 +112,9 @@ public void calculateBrokerHostUsage() {
         SystemResourceUsage usage = new SystemResourceUsage();
         long now = System.currentTimeMillis();
         double elapsedSeconds = (now - lastCollection) / 1000d;
-        double cpuUsage = getTotalCpuUsage(elapsedSeconds);
+        double cpuUsage = (elapsedSeconds <= 0) ? 0 : getTotalCpuUsage(elapsedSeconds);
 
-        if (lastCollection == 0L) {
+        if (lastCollection == 0L || elapsedSeconds <= 0) {
```

Comment:
We could also just add another `if` block here that returns when `elapsedSeconds <= 0` is true. That wouldn't require changing any of the other parts of this method.

Reply:
OK  I  will fix